postmark/core.py

```diff
@@ -338,9 +345,13 @@ def _check_values(self):
             raise PMMailMissingValueException('Cannot send an e-mail without a sender (.sender field)')
         elif not self.__to:
             raise PMMailMissingValueException('Cannot send an e-mail without at least one recipient (.to field)')
-        elif not self.__subject:
+        elif bool(self.__template_id) != bool(self.__template_model):
```

Comment:
Yes, that was a new idiom to me as well. I found it on stackoverflow and though it was pretty cool, so I went with it. But I am happy to change.

What's the best way to change it? Should I change it in my repo? Or do you want to change it after the merge?